i18n/locales/source/dashboard/base.yml

```diff
@@ -947,7 +948,7 @@ en:
     age: Your age
     header: Account Information Update
     warning: "We're in the process of updating our experience to provide you with even stronger privacy protections. To do this, we need existing users to enter their age here."
-    disclaimer: "Code.org will never rent, sell or share this information with any third parties."
+    disclaimer: 'Code.org [will never rent, sell or share this information with any third parties](%{url}).'
```

Comment:
This will cause problems if we keep the old translations, since they will be missing the interpolation pattern.

Reply:
I checked on crowdin and I did not see pre-existing translations for this string.